lms/security.py

```diff
@@ -212,13 +213,24 @@ def _get_lti_jwt(request):
     if not id_token:
         return {}
 
-    try:
-        jwt_params = jwt.decode(id_token, options={"verify_signature": False})
-    except InvalidTokenError as err:
-        raise marshmallow.ValidationError("Invalid id_token", "authorization") from err
+    jwt_service = request.find_service(JWTService)
```

Comment:
There's still a `request.lti_jwt` here which we might move elsewhere later but for now having it here works fine.